third-party/CMakeLists.txt

```diff
@@ -50,6 +50,13 @@ endif()
 
 option(USE_EXISTING_SRC_DIR "Skip download of deps sources in case of existing source directory." OFF)
 
+if(WIN32)
+  find_package(Git)
```

Comment:
@b-r-o-c-k should `find_program` be used instead? https://cmake.org/cmake/help/v3.0/command/find_program.html

Reply:
`find_package` will use [FindGit.cmake](https://github.com/Kitware/CMake/blob/v2.8.7/Modules/FindGit.cmake) which is presumably better because it can find Git installations with non-standard names and paths.